.github/workflows/container-autoupdate.yml

```diff
@@ -2,18 +2,53 @@ name: Build, test and push container
 on:
   schedule:
     - cron: 0 0 * * *
+  # be able to start this action manually from a actions tab when needed
   workflow_dispatch:
 
 jobs:
   push-master-container:
     name: Refresh Rawhide container
     runs-on: ubuntu-latest
+    # start local register to store image locally for testing before pushing to repository
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
     steps:
       - name: Checkout anaconda repository
         uses: actions/checkout@v2
 
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
+        with:
+          driver-opts: network=host
+
+      - name: Build and store image locally for testing
+        uses: docker/build-push-action@v2
```

Comment:
Ugh, why? A plain docker build (preferably `make -f Makefile.am anaconda-ci-build`) already creates it as a local docker image. It's totally unnecessary to involve yet another registry here, it's just needlessly copying around the image on disk.

Also, we really want to use the same command that a developer would do -- namely, `make anaconda-ci-build`. Let's please test this code path as well, and simplify things a lot.